examples/Cargo.toml

```diff
@@ -23,9 +23,9 @@ embedded-hal-bus    = "0.1.0"
 embedded-io-async   = "0.6.1"
 esp-alloc           = "0.3.0"
 esp-backtrace       = { version = "0.11.1", features = ["exception-handler", "panic-handler", "println"] }
-esp-hal             = { version = "0.16.0", path = "../esp-hal" }
+esp-hal             = { version = "0.16.0", path = "../esp-hal", features = ["log"] }
```

Comment:
This prevents us from ever adding a `defmt` example (which would be really since, since that feature is currently completely untested in CI AFAIK). What was the reason for enabling this now?

Reply:
I also often add and remove it ... I use it for logging while implementing something which is very convenient

Maybe we could add a `--log` flag to the xtask's run-example command to avoid setting the feature in Cargo.toml?

Reply:
I see you added a comment regarding this, so I guess I will just ignore it for now :D